.gitignore

```diff
@@ -0,0 +1,14 @@
+# Ignore all
```

Comment:
Because `iverilog` create binary files without extension, use this to ignore files without extension.